great_expectations/rule_based_profiler/data_assistant/data_assistant.py

```diff
@@ -272,6 +274,7 @@ def get_column_standard_deviation_metric_multi_batch_parameter_builder(
         @staticmethod
         def build_metric_multi_batch_parameter_builder(
             metric_name: str,
+            suffix: Optional[str] = None,
```

Comment:
If we always plan to pass `str` or `None` to this method do we need this default?